app/views/docs/functions.phtml

```diff
@@ -503,14 +503,64 @@ static RuntimeResponse &main(const RuntimeRequest &req, RuntimeResponse &res) {
 
 <p><a href="/docs/server/functions#functionsCreateDeployment" rel="noopener"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i> You can also create new code deployments using the Appwrite server API</a></p>
 
+<h3>Manual Deployment</h3>
+<p>You can also upload your functions to be deployed using the Appwrite console. The example below shows a typical Node.JS function, but the same idea applies to any other language.</p>
+
+<div class="ide margin-bottom">
+    <pre class="line-numbers"><code class="prism" data-prism>.
+├── package.json
+└── src
+    └── main.js
```

Comment:
I would stick to `index.js` as suggested name. Just a convention we use everywhere.

Reply:
👀 @Meldiron Should we notify the design team? This isn't what we use in console

Reply:
Ill make sure to discuss it, same inconsistency is there in Open Runtimes.